core/src/scene/drawRule.h

```diff
@@ -36,12 +37,15 @@ class StyleContext;
 
 struct DrawRuleData {
 
+    // https://github.com/tangrams/tangram-docs/blob/gh-pages/pages/draw.md#style-parameters
     std::vector<StyleParam> parameters;
+
+    // https://github.com/tangrams/tangram-docs/blob/gh-pages/pages/draw.md#draw-style
+    // draw-style name (and assigned id)
```

Comment:
This link is outdated now, probably best to replace it with https://github.com/tangrams/tangram-docs/blob/gh-pages/pages/draw.md#draw-rule